### PR TITLE
Add spec file for Fedora distros

### PR DESCRIPTION
**Pull request by danalsan**

### CODE REVIEW

File: packaging/fedora/os-log-merger.spec [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+%{_bindir}/os-log-merger
+%{_bindir}/oslogmerger
+%{_bindir}/netprobe
+
+
+
+%changelog
+* Tue Jul 19 2016 dani - 1.0.6-1
+- First version of the os-log-merger-package
+
```
Comment (mangelajo) [inline packaging/fedora/os-log-merger.spec:57]: nitpicker hat here, there's empty bullets after this :-)

**Comment by mangelajo** [comment]
Wow!, thanks a lot!  

Even documents!!!!

it looks good, can you fix the nit thing I found?

**Comment by mangelajo** [comment]
Thanks a lot!!

**Comment by danalsan** [comment]
Thanks to you :)

On Fri, Jul 22, 2016 at 12:54 PM, Miguel Ángel Ajo <notifications@github.com

**Pull request merged**